src/main/java/by/bcrypto/bee2j/der/DerAnchor.java

```diff
@@ -0,0 +1,13 @@
+package by.bcrypto.bee2j.der;
+
+import com.sun.jna.Pointer;
+import com.sun.jna.Structure;
+
+/*!	\brief Якорь для кодирования контейнеров */
+@Structure.FieldOrder({"der", "pos", "tag", "len"})
+public class DerAnchor extends Structure implements Structure.ByReference {
+	public Pointer der;	    /*!< код */
```

Comment:
Delete pls comments

Reply:
Fixed in 4ae65a024ec0b87e172bca84b932004e5843f5ca